src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests.cs

```diff
@@ -68,10 +68,7 @@ public partial class AddUsingTests : AbstractAddUsingTests
             Workspace workspace, TestParameters parameters)
         {
             var outOfProcess = (bool)parameters.fixProviderData;
-            workspace.Options = workspace.Options.WithChangedOption(RemoteHostOptions.RemoteHostTest, outOfProcess)
-                                                 .WithChangedOption(RemoteFeatureOptions.OutOfProcessAllowed, outOfProcess)
-                                                 .WithChangedOption(RemoteFeatureOptions.AddImportEnabled, outOfProcess);
-
+            workspace.Options = workspace.Options.WithChangedOption(RemoteHostOptions.RemoteHostTest, outOfProcess);
```

Comment:
note: these features fallback to running proc if the oop server doesn't launch.  So it's till good to ahve tests that validate they work properly both in proc and out of proc.